.circleci/config.yml

```diff
@@ -54,6 +54,20 @@ jobs:
           echo "Version from kube-linter: ${version}. Expected version: ${expected_version}"
           [[ "${version}" == "${expected_version}" ]]
 
+    - run:
```

Comment:
Note that I think this should be a separate job in the workflow, but can understand if this feels overkill

Reply:
Yeah, I didn't think it was worth the cruft of persisting and attaching from workspace etc